app/views/step_by_step_pages/index.html.erb

```diff
@@ -6,78 +6,121 @@
   ]
 %>
 
-<%= render 'shared/steps/step_breadcrumb', links: links %>
```

Comment:
Weren't the new breadcrumbs added in the previous commit?

Reply:
We don't need the breadcrumbs on the main index page as it does not provide any value